src/main/java/com/mael/payroll/paymentSchedules/MonthlySchedule.java

```diff
@@ -0,0 +1,4 @@
+package com.mael.payroll.paymentSchedules;
+
+public class MonthlySchedule implements PaymentSchedule {
```

Comment:
As discussed yesterday, this looks odd at the beginning, but it kind of makes sense from the exercise point of view. There are a lot of classes created upfront, but they get more behaviour later on, don't they?